### PR TITLE
Automated cherry pick of #20: Bump go build and get repo ready for new release process

### PR DESCRIPTION
**Pull request by Brian-McM**
Cherry pick of #20 on release-v1.1.

#20: Bump go build and get repo ready for new release process

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Review by doublek — APPROVED** [review]
LGTM

**Pull request merged**
